configuration/email.md

```diff
@@ -39,28 +42,27 @@ Here is an example configuration block which grants access to a few groups and e
 	Allowed-To = `[a-f0-9]+@example\.io`
 ```
 
-
-### Required Shared Email Configuration Parameters
+### Required Parameters
 
 At a minimum, the `[Shared-Mail-Config]` requires the following SMTP parameters:
 
 * `Server`: FQDN of an SMTP server.
 * `Port`: SMTP server port.
-* `Username`: Username used for authenticating with the SMTP server.
-* `Password`: Password used for authenticating with the SMTP server.
 
-### Optional Shared Email Configuration Parameters
+### Optional Parameters
 
-The shared Email configuration also has several optional configuration parameters which can be used to enforce connection behavior, sender addresses, receiver addresses, and Gravwell group access.  The following are optional parameters:
+The shared email configuration also has several optional parameters which can be used to enforce connection behavior, sender addresses, receiver addresses, and Gravwell group access.  The following are optional parameters:
 
-* `UseTLS`: Boolean indicating that StartTLS should be used.  Defaults to false.
+* `Username`: Username used for authenticating with the SMTP server. If unset, Gravwell attempts to send email unauthenticated.
+* `Password`: Password used for authenticating with the SMTP server. If unset, Gravwell attempts to send email unauthenticated.
+* `UseTLS`: Boolean indicating that a STARTTLS command should be issued before authenticating.  Defaults to false.
```

Comment:
I will change Use TLS in the GUI and update this page appropriately, but if we are still calling this UseTLS in the config file, then can we add some extra clarification about this:
- Further stress in documentation that "Use TLS" is for the "StartTLS" command rather than connection encryption. "Use TLS" does not refer to connection encryption (which is what I assumed), but rather simply that the session run the "StartTLS" SMTP command (which is what is documented).

We do clearly state here what it IS, but let's be more explicit about what it IS NOT